cmd/ocm-backplane/login/login.go

```diff
@@ -92,6 +94,14 @@ func init() {
 		"",
 		"Login using PagerDuty incident id or html_url.",
 	)
+	flags.StringVarP(
+		&args.defaultNamespace,
+		"namespace",
+		"n",
+		"default",
+		"Prompts for a  default namespace for SRE to run commands like oc debug node/xxx",
```

Comment:
Let's be more generic here, Backplane isn't used by only SREs :)

```suggestion
		"The  default namespace for a user to execute commands in",
```